src/utils/strings.ts

```diff
@@ -5,3 +5,11 @@ export const toLowerCamelCase = (input: string | undefined) : string => {
 
   return input.charAt(0).toLowerCase() + input.slice(1)
 }
+
+export const toCamelCase = (input: string | undefined) : string => {
```

Comment:
just like with previous comment -  really this just capitalizes first character, does nothing in regards to converting to actual camelCase. I understand that is probably all you need - in that case just change the naming please. If you need actual camelCase use something like lodashes `camelCase` fn (and optionally capitalize the first letter)